### ISSUE TITLE
Improve Parse Server Guide for Email Options, Account Lockout, PasswordPolicy 📕

### ISSUE THREAD

**Issue by dblythy** [ParsePlatform/Docs#808]
__References:__
There are a number of options for Parse Server that are only explained on [github](https://github.com/parse-community/parse-server#email-verification-and-password-reset) but aren't explained at all in the [Parse Server Guide](https://docs.parseplatform.org/parse-server/guide/#getting-started).

__Proposed solution:__
It's my view that the Parse Server Guide should be able to explain all the server options and configuration without readers having to know to go to github. For example, if you search for "Mail options (or similar)" on the docs, nothing shows. It could be assumed that these features don't exist.

On that note, I would propose stripping back the README.md on the repo, transitioning most of the contents to the docs site, and mostly pointing the readme to the docs, such as other packages do, eg [react](https://github.com/facebook/react), [vue](https://github.com/vuejs/vue). I think only basic configuration should be shown on the github, and the rest should point to the docs site.

**Comment by TomWFox** [comment]
I would agree that it makes sense to document all the options (that aren't experimental - unless clearly denoted). I'm definitely in agreement on stripping back the readme, this should make future docs changes more clear IMO.

**Comment by dblythy** [comment]
Maybe a subcategory on the docs page for experimental / untested “at your own risk” features? And then after a period of time transition those features into the core set when we’re confident with them

**Comment by TomWFox** [comment]
Yeah that would probably be good.
